src/utils/handlers/general/checkQuery.ts

```diff
@@ -27,7 +27,7 @@ export function checkQuery(string: string): QueryData {
     } else if (/youtube|youtu\.be/g.exec(url.hostname)) {
         result.sourceType = "youtube";
 
-        if (!/youtu\.be/g.exec(url.hostname) && url.pathname.startsWith("/playlist")) {
+        if (!/youtu\.be/g.exec(url.hostname) && url.pathname.startsWith("/playlist") || url.href.includes('list=')) {
```

Comment:
You can use `url.searchParams.has("list")` instead to check whether there's `list` query or not